components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -98,6 +99,13 @@ export const PrebuildDetailPage: FC = () => {
         };
     }, [prebuildId, disposeStreamingLogs]);
 
+    useEffect(() => {
+        const anyLogAvailable = prebuild?.status?.taskLogs.some((t) => t.logUrl);
```

Comment:
Not sure if `prebuild` status have all `taskLogs` after init, maybe we need a watch here, how about use `currentPrebuild`? as it's already watched

Reply:
And need another logic to set it false. i.e. Changed to another prebuild that has log

Reply:
If we change prebuild (i.e. trigger a new one), the error does get reset already - [ref](https://github.com/gitpod-io/gitpod/blob/221a1a1f99520790ad95858d483c33251a855bcc/components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx#L89). It makes sense to use `currentPrebuild` here. Maybe it also makes sense to use `currentPrebuild` here? WDYT? https://github.com/gitpod-io/gitpod/blob/221a1a1f99520790ad95858d483c33251a855bcc/components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx#L91